### ISSUE TITLE
error parsing json and php array

### ISSUE THREAD

**Issue by IsaevDimka** [innocenzi/laravel-vue-lang#3]
WARNING in ./resources/lang/en.json
Module parse failed: Unexpected token m in JSON at position 0 while parsing near 'module.exports = {"S...'
File was processed with these loaders:
  ./node_modules/laravel-localization-loader/index.js
You may need an additional loader to handle the result of these loaders.
SyntaxError: Unexpected token m in JSON at position 0 while parsing near 'module.exports = {"S...'
    at JSON.parse (<anonymous>)
    at parseJson (/var/www/laravel/node_modules/json-parse-better-errors/index.js:7:17)
    at JsonParser.parse (/var/www/laravel/node_modules/webpack/lib/JsonParser.js:16:16)
    at /var/www/laravel/node_modules/webpack/lib/NormalModule.js:482:32
    at /var/www/laravel/node_modules/webpack/lib/NormalModule.js:358:12
    at /var/www/laravel/node_modules/loader-runner/lib/LoaderRunner.js:373:3
    at iterateNormalLoaders (/var/www/laravel/node_modules/loader-runner/lib/LoaderRunner.js:214:10)
    at iterateNormalLoaders (/var/www/laravel/node_modules/loader-runner/lib/LoaderRunner.js:221:10)
    at /var/www/laravel/node_modules/loader-runner/lib/LoaderRunner.js:236:3
    at runSyncOrAsync (/var/www/laravel/node_modules/loader-runner/lib/LoaderRunner.js:130:11)
    at iterateNormalLoaders (/var/www/laravel/node_modules/loader-runner/lib/LoaderRunner.js:232:2)
    at Array.<anonymous> (/var/www/laravel/node_modules/loader-runner/lib/LoaderRunner.js:205:4)
    at Storage.finished (/var/www/laravel/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:55:16)
    at /var/www/laravel/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:91:9
    at /var/www/laravel/node_modules/graceful-fs/graceful-fs.js:115:16
    at FSReqCallback.readFileAfterClose [as oncomplete] (internal/fs/read_file_context.js:63:3)
 @ ./resources/lang sync \.(php|json)$ ./en.json
 @ ./node_modules/laravel-vue-lang/dist/catalogue.js
 @ ./node_modules/laravel-vue-lang/dist/index.js
 @ ./resources/js/localization.js
 @ ./resources/js/app.js
 @ multi ./resources/js/app.js ./resources/sass/main.scss

Example en.json

```json
{
    "Sign In": "Sign In",
    "Remember Me": "Remember Me",
    "Forgot password": "Forgot password",
    "New Account": "New Account"
}
`

**Comment by ttrushin** [comment]
I am suddenly getting this error as well, however things were working great when I was only using PHP lang files. Issue seems to be with JSON files.

**Comment by hawezo** [comment]
Hi, this package is a wrapper around `Lang.js` and [`laravel-localization-loader`](https://github.com/rmariuzzo/laravel-localization-loader). 

I suspect the issue is coming from the latter, so you may get more help on that repository. If you think the issue is actually coming from this package, please tell me why and I'll take a look at it. Thanks!

**Comment by rvbz** [comment]
Pull request #7 would help to fix this issue. 👍

**Comment by innocenzi** [comment]
I'm on it right now. 👍

**Comment by innocenzi** [comment]
This should be fixed with 727bb9b37acfcdca1fce5f6dd37368dfbc36728b.
